x/pki/client/cli/query_approved_certificates.go

```diff
@@ -12,6 +12,7 @@ import (
 )
 
```

Comment:
1) I would keep `all-x509-certs` untouched (so that it returns all certs with pagination as other `all-xxx` commands).
I would update the `x509-cert` command to make `subject` optional there, so that if subject is specified - existing logic is called (get certificates by subject+subject-key-id), if `subject` is not specified - then all certificates by subject-key-id are returned.
2) Please make sure that get by subject-key-id works with light client approach by using `cli.QueryWithProof` instead of `types.QueryAllApprovedCertificatesRequest`.